WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -862,59 +858,6 @@ private extension ProductsViewController {
             }
         }
     }
-
-    func observeBlazeBannerVisibility() {
-        viewModel.$shouldShowBlazeBanner
-            .removeDuplicates()
-            .combineLatest($dataLoadingError)
-            .sink { [weak self] shouldShow, loadingError in
-                guard let self else { return }
-                if shouldShow, loadingError == nil {
-                    self.showBlazeBanner()
-                } else {
-                    self.hideBlazeBanner()
-                }
-            }
-            .store(in: &subscriptions)
-    }
-
-    func updateBlazeBannerVisibility() {
-        Task { @MainActor in
-            await viewModel.updateBlazeBannerVisibility()
-        }
-    }
-
-    func showBlazeBanner() {
-        guard blazeBannerHostingController == nil else {
```

Comment:
We can remove this variable as we don't use it anymore. 

```swift
private var blazeBannerHostingController: BlazeBannerHostingController?
```

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/10960/commits/1d317e12c23f7280d476608eac1af204e9c7f642